### DISCUSSION TITLE
pgbouncer cannot connect to server

### DISCUSSION THREAD

**Discussion by Korayem** [supabase/supabase#8832] [Questions]
Everything was working fine for past few months

Just now, I am getting `pgbouncer cannot connect to server`

I am connected to PORT 6543 and didn't change anything

I already sent to support but they take 24-48 hours to respond

**Comment by Korayem** [comment]
I did try to restart the database but same issue

Even on supabase dashboard I am getting `Error: [500] failed to get pg.tables: Connection terminated unexpectedly` when I try to look at the tables

**Reply by davbauer** [reply]
Any update on this one, I am getting the same error.
